### DISCUSSION TITLE
Смена голоса озвучки

### DISCUSSION THREAD

**Discussion by h0rt** [ilyhalight/voice-over-translation#250] [Q&A] [answered]
Доброго времени суток!
Можно ли как-то изменить голос, поменять приоритет голоса озвучки?

**Comment by NullVerdict** [comment, marked as answer]
> Доброго времени суток! Можно ли как-то изменить голос, поменять приоритет голоса озвучки?

Нет, у Яндекса это все делается автоматически, и вы ничего не можете изменить. Вы можете попросить Яндекс добавить такую возможность.
